<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LimitConv</class>
 <widget class="QDialog" name="LimitConv">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>469</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>923</width>
    <height>469</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>923</width>
    <height>469</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TKRtoTKR3</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>861</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnFile">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>20</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnSave">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>430</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>save as TKR3</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>901</width>
     <height>361</height>
    </rect>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::DragDrop</enum>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>50</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Type</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Title (old)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Title (new)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>SerialNo</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>CalDate</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Comment</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="BtnDel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete item</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnCancel">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>430</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnOk">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>430</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
